scripts/README.md

```diff
@@ -0,0 +1,25 @@
+# Publish and Tag Docker Image
+
+The `publish-docker.sh` script does the following interactively:
+
+- Ask which packages to  include (always answer by SPACE-RETURN)
+- Ask which packages get a major version bump
```

Comment:
I interpret major version bump as changing the first number, so going:
from 0.3.0 to 1.0.0
from 1.0.0 to 2.0.0

If I do not indicate a package as needing a major version bump would the second or the third number change? Will it go from 0.3.0 to 0.3.1 or to 0.4.0?
Is there a way to specify the version number or indicate if the first, second, or third number should be bumped?